CHANGELOG.md

```diff
@@ -2,6 +2,10 @@ haproxy Cookbook CHANGELOG
 ==========================
 This file is used to list changes made in each version of the haproxy cookbook.
 
+v1.9.7
+------
+- Lawrence: enable 1.5 admin features
```

Comment:
But doesn't your commit already tie the change to you? That was more my point. This would only be needed in the terrible world of not maintaining modified community cookbooks in source control.